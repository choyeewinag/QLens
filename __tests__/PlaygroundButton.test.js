import React, { useState, useEffect } from 'react';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import PlaygroundButton from '../src/Components/PlaygroundButton';

//Basic Test with React-test-renderer
it('renders correctly react-test-renderer', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<PlaygroundButton />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });