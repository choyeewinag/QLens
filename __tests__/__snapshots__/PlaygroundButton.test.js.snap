// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div
  className="playgroundButton"
>
  <button
    className="btn draw-border"
  >
    <a
      href="http://localhost:3000/graphql"
      target="_blank"
    >
      Playground
    </a>
  </button>
</div>
`;
